src/core/adapters/keycloakOidcClient.ts

```diff
@@ -72,6 +72,7 @@ export async function createKeycloakOidcClient(params: {
     const oidcClient = id<OidcClient.LoggedIn>({
         "isUserLoggedIn": true,
         "accessToken": keycloakInstance.token!,
+        "refreshToken": keycloakInstance.refreshToken!,
```

Comment:
I'm pretty sure that when the accessToken is refreshed, the refreshToken changes as well.  
Also, you should pull, this have been updated. It's now `getToken()`